src/literals.py

```diff
@@ -16,7 +16,11 @@
 REL_NAME = "kafka-client"
 INTER_BROKER_USER = "sync"
 ADMIN_USER = "admin"
+EXPORTER_USER = "exporter"
```

Comment:
Mykola made a point about this being Prometheus specific. `monitoring` name is a better choice, as it can also be reused by other observability pieces in the future.